scripts/save.sh

```diff
@@ -252,6 +252,15 @@ dump_bash_history() {
 		done
 }
 
+remove_old_backups() {
+	# remove backup files older than 30 days, but keep at least 5 copies of backup.
+	local -a files
+	pushd "$(resurrect_dir)" >/dev/null
+	files=($(ls -t ./*.txt | tail -n +6))
```

Comment:
Is there any benefit of using `pushd` + `popd` over just `ls -t $(resurrect_dir)/*.txt`